### ISSUE TITLE
[terminal] *Nushell* support

### ISSUE THREAD

**Issue by stevenxxiu** [albertlauncher/plugins#97]
#### Description
Commands do not run under *Nushell*.

#### Steps to reproduce
In order to run a command, say in this instance `top`, *Albert Launcher* runs:

```
$ nu -ic top ; exec /usr/bin/nu
Error: nu::parser::short_flag_arg_cant_take_arg (link)

  × Batches of short flags can't take arguments.
   ╭─[source:1:1]
 1 │ nu -ic
   ·      ┬
   ·      ╰── short flag batches can't take args
   ╰────

```

As can be seen, this does not work.

In order to run a binary using *Nushell*, **and** include the user's `$PATH` settings (aka loading the users configured `env.nu`), the only way I know is by:

    $ nu --execute top

Respecting `$PATH` is important, as otherwise `top` cannot be aliased to an alternative. I believe most people have their own preferred `top` that isn't the default `procps-ng` one.

So this always has to launch an interactive shell.

#### Source
Built from source.

**Comment by ManuelSchneid3r** [comment]
> So this always has to launch an interactive shell.

This is why there is the `-i` option. I am not sure if I got things right, but to me this looks like nushell is simply not bourne shell compatible. You could start a discussion with nushell devs and ask why they dont strive for compatibility. So far all shells I came across support -i and -c.

**Comment by ManuelSchneid3r** [comment]
<img width="708" alt="Bildschirm­foto 2023-03-06 um 10 45 51" src="https://user-images.githubusercontent.com/3252315/223074655-3322401a-b2df-4271-9e90-8e8bd122dedd.png">

Acutally they are. Can you rephrase the problem in case I missed the point, or otherwise maybe you should open an issue at nushell, asking why they dont use $PATH in interactive mode.

**Comment by stevenxxiu** [comment]
I can look at the `$PATH` issue. But would changing this to `-i -c` instead be reasonable? *Nushell* works that way.
